### PR TITLE
Homework

### PR DESCRIPTION
**Pull request by svirsky**

### CODE REVIEW

**Comment by svirsky** [comment]
Homework

File: src/com/rakovets/course/practice/module2/Task04.java [hunk 1]
```diff
@@ -16,10 +16,13 @@ public static void main(String[] args) {
 		int weight = 9;
 
 		// Код необходимый для тестирования, не изменять
-		weight = (args.length == 0) ? weight : Integer.parseInt(args[0]);
+		weight = (args.length == 0) ? weight : Integer.valueOf(args[0]);
 
 		//TODO
 		// Код, решающий задачу пишем ниже, при этом используяся переменные объявленные выше (их можно изменять)
 		// Для проверки решения необходимо запустить @Test для данного class (в директории test)
+
```
Comment (rakovets) [inline src/com/rakovets/course/practice/module2/Task04.java]: INFO: violation of code convention

File: src/com/rakovets/course/practice/module2/Task06.java [hunk 2]
```diff
@@ -16,11 +16,14 @@ public static void main(String[] args) {
...
 		// Код необходимый для тестирования, не изменять
-		deposit = (args.length == 0) ? deposit : Integer.parseInt(args[0]);
-		annualPercentage = (args.length == 0) ? annualPercentage : Integer.parseInt(args[1]);
+		deposit = (args.length == 0) ? deposit : Integer.valueOf(args[0]);
+		annualPercentage = (args.length == 0) ? annualPercentage : Integer.valueOf(args[1]);
 
 		//TODO
 		// Код, решающий задачу пишем ниже, при этом используяся переменные объявленные выше (их можно изменять)
 		// Для проверки решения необходимо запустить @Test для данного class (в директории test)
+
```
Comment (rakovets) [inline src/com/rakovets/course/practice/module2/Task06.java]: INFO: violation of code convention

File: src/com/rakovets/course/practice/module2/Task07.java [hunk 3]
```diff
@@ -14,10 +14,16 @@ public static void main(String[] args) {
 		int diameter = 10;
 
 		// Код необходимый для тестирования, не изменять
-		diameter = (args.length == 1) ? Integer.parseInt(args[0]) : diameter;
+		diameter = (args.length == 1) ? Integer.valueOf(args[0]) : diameter;
 
 		//TODO
 		// Код, решающий задачу пишем ниже, при этом используяся переменные объявленные выше (их можно изменять)
 		// Для проверки решения необходимо запустить @Test для данного class (в директории test)
+
```
Comment (rakovets) [inline src/com/rakovets/course/practice/module2/Task07.java]: INFO: violation of code convention

File: src/com/rakovets/course/practice/module2/Task07.java [hunk 4]
```diff
@@ -14,10 +14,16 @@ public static void main(String[] args) {
...
+		diameter = (args.length == 1) ? Integer.valueOf(args[0]) : diameter;
 
 		//TODO
 		// Код, решающий задачу пишем ниже, при этом используяся переменные объявленные выше (их можно изменять)
 		// Для проверки решения необходимо запустить @Test для данного class (в директории test)
+
+		int radius = diameter / 2;
+		double S = Math.PI * Math.pow(radius, 2);
+		System.out.println(S);
+
```
Comment (rakovets) [inline src/com/rakovets/course/practice/module2/Task07.java]: INFO: violation of code convention

File: src/com/rakovets/course/practice/module2/Task07.java [hunk 5]
```diff
@@ -14,10 +14,16 @@ public static void main(String[] args) {
...
+
```
Comment (rakovets) [inline src/com/rakovets/course/practice/module2/Task07.java]: INFO: violation of code convention

File: src/com/rakovets/course/practice/module2/Task10.java [hunk 6]
```diff
@@ -16,11 +16,14 @@ public static void main(String[] args) {
...
-		height = (args.length == 2) ? Integer.parseInt(args[0]) : height;
-		weight = (args.length == 2) ? Integer.parseInt(args[1]) : weight;
+		height = (args.length == 2) ? Integer.valueOf(args[0]) : height;
+		weight = (args.length == 2) ? Integer.valueOf(args[1]) : weight;
 
 		//TODO
 		// Код, решающий задачу пишем ниже, при этом используяся переменные объявленные выше (их можно изменять)
 		// Для проверки решения необходимо запустить @Test для данного class (в директории test)
+		int standartVes = height - 110;
+		int izbitVes = weight - standartVes;
```
Comment (rakovets) [inline src/com/rakovets/course/practice/module2/Task10.java]: INFO: bad naming

File: src/com/rakovets/course/practice/module2/Task11.java [hunk 7]
```diff
@@ -17,10 +17,14 @@ public static void main(String[] args) {
 		double temperatureInCelsius = 0;
 
 		// Код необходимый для тестирования, не изменять
-		temperatureInCelsius = (args.length == 1) ? Double.parseDouble(args[0]) : temperatureInCelsius;
+		temperatureInCelsius = (args.length == 1) ? Double.valueOf(args[0]) : temperatureInCelsius;
 
 		//TODO
 		// Код, решающий задачу пишем ниже, при этом используяся переменные объявленные выше (их можно изменять)
 		// Для проверки решения необходимо запустить @Test для данного class (в директории test)
+		double tempFareng = temperatureInCelsius * 1.8 + 32;
```
Comment (rakovets) [inline src/com/rakovets/course/practice/module2/Task11.java]: INFO: bad naming

**Comment by rakovets** [comment]
Not bad ^_^

**Pull request closed without merging**